### PR TITLE
docs(authentication): Added section about authentication methods

### PR DESCRIPTION
**Pull request by vzpgb**

### CODE REVIEW

File: content/resources/operator/_index.md [hunk 1]
```diff
@@ -81,7 +81,106 @@ Blood Pressure Reading:
...
+* User’s organization does not inherently take responsibility for actions taken by user under auspice of ORCID identity.
+* Neither user’s organization nor ORCID take responsibility for activating/deactivating users based on affiliation.
+
+### Google / Microsoft Office 365 (personal account)
+
+Google Sign-In / Microsoft Office 365 secure authentication systems as registered by an individual with no organizational affiliation or management.
+
+_Pros_
+
+* Most users either have an Google/Microsoft identities or can easily create one.
```
Comment (cgmeyer) [inline content/resources/operator/_index.md]: ...have a ... identity...
...have Google/Microsoft identities ...

File: content/resources/operator/_index.md [hunk 2]
```diff
@@ -81,7 +81,106 @@ Blood Pressure Reading:
...
 
-## 4. Programs and Projects
+
+## 4. Authentication Methods
+
+The following methods of authentication are supported in Gen3. They are listed in order of preference from OCC perspective, and technological and governance considerations are outlined below.
+
+### eRA Commons
+
+Authentication system developed by NIH for managing for research grants.
```
Comment (abgeorge7) [inline content/resources/operator/_index.md]: I think it should be “for managing of research grants” or “for the management of research grants”

File: content/resources/operator/_index.md [hunk 3]
```diff
@@ -81,7 +81,106 @@ Blood Pressure Reading:
...
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+
+_Cons_
+
+* Can only be used if the user’s organization is part of the InCommon federation. 
+* Reference this resource for organizations supported by InCommon.
```
Comment (abgeorge7) [inline content/resources/operator/_index.md]: Maybe I’m missing something, what is the con here?

File: content/resources/operator/_index.md [hunk 4]
```diff
@@ -81,7 +81,106 @@ Blood Pressure Reading:
...
+* International presence, and InCommon is a participant
+
+_Cons_
+
+* Can only be used if the user’s organization is part of one of the 60+ federations participating in eduGAIN. 
+* Reference this resource for organizations supported by eduGAIN.
+
+### ORCID
+
+Provide an identifier for individuals to use with their name as they engage in research, scholarship, and innovation activities.
```
Comment (abgeorge7) [inline content/resources/operator/_index.md]: “Provides an identifier...”

Comment (vzpgb) [reply content/resources/operator/_index.md]: Thanks, fixed!

File: content/resources/operator/_index.md (hunk 1 above)
Comment (vzpgb) [reply content/resources/operator/_index.md]: Thanks! Done in 3d1dc98.

File: content/resources/operator/_index.md (hunk 2 above)
Comment (vzpgb) [reply content/resources/operator/_index.md]: Thanks! Done in 02e5565.

File: content/resources/operator/_index.md (hunk 3 above)
Comment (vzpgb) [reply content/resources/operator/_index.md]: Not you, I'm missing links! Fixed in e59cb35

**Pull request merged**